amplify/backend/function/fetchDomainBalance/src/utils.js

```diff
@@ -117,12 +117,15 @@ const buildAPIEndpoint = (url, queryParams) => {
 const getStartOfDayFor = (date) =>
   formatISO(new Date(startOfDay(new Date(date))));
 
-const getFormattedIncomingFunds = (incomingFunds, parentDomainId) =>
+const getFormattedIncomingFunds = (incomingFunds, domainId) =>
   incomingFunds.map((incomingFund) => ({
     amount: incomingFund.amount,
     finalizedDate: incomingFund.updatedAt,
     token: incomingFund.token,
-    toDomainId: parentDomainId,
+    // @TODO: There will only ever be incomingFunds if "All teams" or the "Root domain" is selected
+    // So this value will only ever be undefined or 1
+    // Not sure if it is actually needed?
```

Comment:
It's only needed to be able to properly add it to the ` balance[period].out` or ` balance[period].in` array